docs/quickstart.md

```diff
@@ -0,0 +1,72 @@
+# Quick Start:
+
+The most basic Spiel presentation looks like this:
+
+```python
+from spiel import Deck, present
+
+deck = Deck(name="Your Deck Name")
+
+@deck.slide(title="Slide 1 Title")
+def slide1():
+    content = """\
+Your content here
+    """
+    return content
+
+if __name__ == "__main__":
+    present(__file__)
+```
+
+And when you run the Python file:
+
+![Barebones slide](./assets/quickstart1.svg)
+
+To recap, you first create a Deck object that has the name of your presentation. Then you create slide functions, add content into a triple-quoted string (aka a multiline string) and return it, and add the slide to the Deck with the `@deck.slide()` decorator. Finally, you call `present()` to generate the presentation.
+
+You can make your slides a lot prettier, of course. Spiel renders its slides using Rich, so you can bring in Rich functionality to spruce up your slides. For instance, to set up a slide with a left side and right side (as a code demonstration, maybe), let's bring in some Rich features:
+
+``` python
+from rich.console import RenderableType
+from rich.layout import Layout
+```
+
+And now let's use those in our slide:
+
+```python
+@deck.slide(title="Slide 1 Title")
+def slide1() -> RenderableType:
+    left_side = f"""\
+I'm a code sample, and I'm on the left!
+
+print("Hello World!")
+        """
+
+    right_side = f"""\
+And I'm the code sample output, and I'm on the right!
+
+Hello World! # *see note below
+        """
+
+    root = Layout()
+
+    root.split_row(
+        Layout(left_side),
+        Layout(right_side),
+    )
+
+    return root
+```
+
+And when you run the Python file:
+
+![Barebones split slide](./assets/quickstart2.svg)
+
+
+*Note: There are ways to execute code and get the output in your string, but those are beyond the scope of this guide. Check out [this Stack Overflow post](https://stackoverflow.com/questions/3906232/python-get-the-print-output-in-an-exec-statement) as a starting point if you are curious.
```

Comment:
Executable code examples was actually pretty tricky to implement for https://github.com/JoshKarpel/pytest-its-whats-for-testing and I would consider it (currently) a non-standard feature 😅 . Though it's such an obvious direction to go with the tool that I should probably figure out a way to roll it in first-class...

Point is - maybe some other example here? I've always thought that this https://github.com/JoshKarpel/spiel/blob/6321c01418a0d0b262042ef9a555e652803c4772/spiel/demo/demo.py#L122-L156 demo slide was a good simple example, showing off both fancy Rich rendering with layouts and also the introspection capabilities of running inside Python.